generic/tests/mac_change.py

```diff
@@ -113,6 +114,7 @@ def run(test, params, env):
             if os_variant == "winxp":
                 connection_id = pnpdevice_id.split("&")[-1]
                 mode = "devcon"
+            time.sleep(10)
```

Comment:
None is returned from wait_for() instead of utils_net.restart_windows_guest_network(), which is as expected.
Just only judge the return value of wait_for() is fine.

if not wait_for():
    raise error....

But in my test, I always met error which blocks my test.
15:01:49 ERROR| ShellStatusError: Could not get exit status of command: 'netsh interface set interface name="Ethernet" admin=DISABLED'    (output: 'C:\\Windows\\system32>\n')

@xutian ,
What's your opinion?